classes/DefinitionItem.php

```diff
@@ -101,7 +101,7 @@ public static function initFromArray($items)
      */
     public function getTypeOptions()
     {
-        $result = ['url' => 'URL'];
+        $result = ['url' => trans('winter.sitemap::lang.item.url')];
```

Comment:
In pure PHP files we prefer to use `Lang::get()` instead of `trans()` just to make it more clear what's going on. View files are fine to use `trans()` though. Please swap this one and import `Lang` at the top. Thanks for your contribution!

Reply:
I changed it.

Reply:
Perfect, thanks!